### PR TITLE
Applied question page title style to calculated summary page title

### PR DESCRIPTION
**Pull request by pricea1**
### What is the context of this PR?
Calculated Summary Title is not applying highlight styles.

### How to review 
Add a few pages with number answers.
Add a calculated summary page.
In the title of the calculated summary page, use the rich text editor to highlight some of the content.
Preview/launch the questionnaire.
Observe that the highlighting is visible on the calculated summary page.

### CODE REVIEW

**Comment by pricea1** [comment]
https://github.com/ONSdigital/eq-author-app/issues/343

**Comment by pricea1** [comment]
[Highlighting in title not shown on calculated summary page on runner](https://trello.com/c/O3HHKy2o/1120-highlighting-in-title-not-shown-on-calculated-summary-page-on-runner)

File: app/templates/summary.html [hunk 1]
```diff
@@ -30,15 +30,15 @@ <h2 class="u-fs-l">{{ _("Please remember to submit these answers") }}.</h2>
 {% elif content.summary.summary_type in ('SectionSummary', 'AnswerSummary') %}
 <div>
     {% set group = content.summary.groups %}
-    <h1 class="u-fs-l">{{content.summary.title}}</h1>
+    <h1 class="u-fs-l question__title">{{content.summary.title}}</h1>
```
Comment (samiwel) [inline app/templates/summary.html]: @hamishtaplin is it okay to use this style on the summary page? Seems to fix the problem, but semantically this is a summary title not a question title? 🤷‍♂

Comment (hamishtaplin) [reply app/templates/summary.html]: Not really. Survey Runner follows [The BEM convention](http://getbem.com/naming/) of writing CSS/components and so `question__title` should only be used within the scope of a `question` block.

Either we don't support it here, or we should make a CSS change.

Comment (pricea1) [reply app/templates/summary.html]: Now changed so its within scope of `question` block

**Comment by samiwel** [comment]
IMO I think we're better off reverting this change. They are trying to highlight in a summary title which survey runner doesn't support. Rather we should probably make a change to Author to disable the highlighting button on the toolbar for summary titles.

**Review by samiwel — APPROVED** [review]
Tested locally and this change styles the title as expected.

**Review by hamishtaplin — APPROVED** [review]

**Pull request merged**